p2p/switch.go

```diff
@@ -583,11 +641,10 @@ func (sw *Switch) acceptRoutine() {
 // add the peer.
 // if dialing fails, start the reconnect loop. If handhsake fails, its over.
```

Comment:
While at it:
```suggestion
// if dialing fails, start the reconnect loop. If handshake fails, it's over.
```
or: it stops instead of the dramatic "it's over"